snapcraft.yaml

```diff
@@ -7,6 +7,76 @@ type: gadget
 architectures:
   - armhf
 confinement: strict
+grade: stable
+parts:
+  uboot:
+    plugin: make
+    source: git://git.denx.de/u-boot.git
+    source-branch: v2017.05
+    prepare: |
+      git apply ../../../uboot.patch
+      make rpi_3_32b_defconfig
+    build: |
+      if [ "$(arch)" = "x86_64" ]; then
```

Comment:
from coreutils (usually available on all build environments, but if you insist i can add it to build-packages as safety net)